wiki/Version-4.0-User-Upgrade-Guide.md

```diff
@@ -97,7 +97,7 @@ To continue running code style and code quality checks on JS and/or CSS files, u
 There are two kinds of sniffs which are affected by this:
 1. Sniffs which are specific to JS/CSS and only target JS/CSS code.  
     If you use JS/CSS specific sniffs from either PHP_CodeSniffer itself or from an external standard, remove references to these sniffs from your ruleset.
-2. "Mixed" sniffs, i.e. sniffs which target both PHP as well as JS and/or CSS code.  
+2. "Mixed" sniffs, i.e., sniffs which target both PHP as well as JS and/or CSS code.  
```

Comment:
The docs are in British English and a comma after `i.e.` is American English, so nope.

Reply:
Ok, I was not aware of this difference between British and American English.